src/AspNetCore/aspnetcore_schema.xml

```diff
@@ -25,6 +25,7 @@
         <attribute name="processesPerApplication" type="uint" defaultValue="1" validationType="integerRange" validationParameter="1,100"/>
         <attribute name="forwardWindowsAuthToken" type="bool" defaultValue="true" />
         <attribute name="disableStartUpErrorPage" type="bool" defaultValue="false" />
+        <attribute name="hostingModel" type="string" defaultValue="outofprocess" />
```

Comment:
can you not have a default? e.g. default to empty? Let's handle the default in code.